plugins/outputs/influxdb/http.go

```diff
@@ -208,6 +208,7 @@ func (c *httpClient) Database() string {
 // Note that some names are not allowed by the server, notably those with
 // non-printable characters or slashes.
 func (c *httpClient) CreateDatabase(ctx context.Context, database string) error {
+	//nolint:gocritic // sprintfQuotedString - "%s" used by purpose, string escaping is done by special function
 	query := fmt.Sprintf(`CREATE DATABASE "%s"`, escapeIdentifier.Replace(database))
```

Comment:
Out of curiosity, what happens if you use `%q` here? Shouldn't this just work?

Reply:
@srebhan Test will fail:
![image](https://github.com/influxdata/telegraf/assets/17336997/03892c0d-3278-44d4-8654-430f66851955)